docs/se/security/05_extensibility.adoc

```diff
@@ -129,4 +129,4 @@ the integration component attempts to authenticate the request.
 
 Because other components of Helidon Security (such as ABAC validators) query the request for annotations,
 the integration component should also collect all annotations from the resource and correctly configure
-them when creating the security request.
+them when creating the security request.
```

Comment:
This looks like a white space change. This should be displayed in the diff as "File renamed without changes"